snippets/python-snippets.json

```diff
@@ -192,39 +192,63 @@
         ],
         "description": "Add onchange field"
     },
-    "Constrains on field": {
+    "SQL Constraints": {
+        "prefix": "oofconstrains",
+        "body": [
+            "_sql_constraints = [",
+            "\t(\"${1:field_name}_${2|check,unique|}\", \"${2}(${1})\", \"${3:Error message}\"),",
+            "]"
+        ],
+        "description": "Add SQL constraints to check or unique field"
+    },
+    "Python Constrains": {
         "prefix": "oofconstrains",
         "body": [
             "@api.constrains('${1:fieldname}')",
             "def _constrains_${1}(self):",
             "\t${0:pass}"
         ],
-        "description": "Add constrains on field"
+        "description": "Add a python constrains"
+    },
+    "Ensure one record": {
+        "prefix": "ooone",
+        "body": [
+            "self.ensure_one()"
+        ],
+        "description": "Ensure one record passing to this method"
+    },
+    "Super hook method": {
```

Comment:
Would be good if you call it as `override`.

Reply:
called as override